src/me/coley/recaf/ui/component/list/OpcodeCellRenderer.java

```diff
@@ -69,7 +69,16 @@ public String getOpcodeText(OpcodeList list, AbstractInsnNode ain) {
 		case AbstractInsnNode.TYPE_INSN:
 			// Add type name to string
 			TypeInsnNode insnType = (TypeInsnNode) ain;
-			String typeDeclaredStr = getTypeStr(Type.getType(insnType.desc), options);
+      Type typeDeclared = null;
+      String typeDeclaredStr = "";
+      try{
+        typeDeclared = Type.getType(insnType.desc);
+			  typeDeclaredStr = getTypeStr(typeDeclared, options);
+      }
+      catch(Exception e)
+      {
+        typeDeclaredStr = insnType.desc;
+      }
```

Comment:
No issues here, but curious to cases where this fails. Do you have any samples you can share that cause this? It may be solve-able without a try-catch if there's some sort of pattern to the failure.